protocol/network-types/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+authors = ["ChainSafe Systems <info@chainsafe.io>"]
+name = "mina-network-types"
```

Comment:
The serialized bytes can also be used for storage, maybe `serializable-types` is better than `network-types`?